lefthook.yml

```diff
@@ -0,0 +1,7 @@
+pre-commit:
+  commands:
+    tapioca-gem:
+      glob: "Gemfile.lock"
+      run: |
+        echo "Changes were detected in Gemfile.lock. Running `tapioca gem`"
+        bin/tapioca gem --verify && bin/tapioca gem
```

Comment:
(this doesn't actually work on `tapioca` itself since it needs to be `bundle exec tapioca`).